test/doc/algo/inclusive_scan.cpp

```diff
@@ -0,0 +1,21 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include "detail/print.hpp"
+
+int main()
+{
+  std::vector<float> r1(5) , r2(5);
+  eve::algo::fill(r1, 2.0f);
+  eve::algo::fill(r2, 2.0f);
+
+  eve::algo::inclusive_scan_inplace(r1, std::pair{eve::add, eve::zero}, 0.0f);
+  eve::algo::inclusive_scan_inplace(r2, std::pair{eve::mul, 1.0f}, 1.0f);
+
+  std::cout << " <- eve::algo::inclusive_scan_inplace(r1, std::pair{eve::add, eve::zero}, 0.0f);\n";
```

Comment:
`std::pair{eve::add, eve::zero}` is implicit, please remove in one place:
  eve::algo::inclusive_scan_inplace(r, 0.f);

https://godbolt.org/z/Khqr3Gand

Reply:
done